portal/src/graphql/portal/Authenticated.tsx

```diff
@@ -122,11 +173,79 @@ export interface ConfigureAuthgearOptions {
 export async function configureAuthgear(
   options: ConfigureAuthgearOptions
 ): Promise<void> {
+  // eslint-disable-next-line no-console -- Output the session type to console for easier debugging.
+  console.log("authgear: sessionType =", SESSION_TYPE);
```

Comment:
console.debug or console.info can be used to avoid lint error, and also give it a proper log level

Reply:
Currently, we only allow `warn` and `error`. I change this call to `console.info` and keep the `no-console` exception.